web/pwa/src/features/former/Field.tsx

```diff
@@ -19,7 +19,7 @@ type FormerFieldProps = {
     setFieldIsKey: (isKey: boolean) => void
     fieldDescription: string,
     setFieldDescription: (description: string) => void
-    openSubForm: () => void
+    openSubForm: (fieldName: string) => void
```

Comment:
You should not need the field name here..

Reply:
Alright, also to note, the server choked up on nested forms going more than 1 level deep
ie: Parent -> Child = OK
Parent -> Child -> Grandchild -> ... = NOT OK

Reply:
This is not resolved yet, the fieldName should not be needed as we can infer it from the fieldId.